src/plugins/reporter-bugzilla.c

```diff
@@ -520,28 +554,11 @@ int main(int argc, char **argv)
         free_report_result(reported_to);
     }
 
-    problem_data_t *problem_data = create_problem_data_for_reporting(dump_dir_name);
-    if (!problem_data)
-        xfunc_die(); /* create_problem_data_for_reporting already emitted error msg */
-
     const char *component = problem_data_get_content_or_die(problem_data, FILENAME_COMPONENT);
     const char *duphash   = problem_data_get_content_or_NULL(problem_data, FILENAME_DUPHASH);
 //COMPAT, remove after 2.1 release
     if (!duphash) duphash = problem_data_get_content_or_die(problem_data, "global_uuid");
 
-    if (!rhbz.b_product || !*rhbz.b_product) /* if not overridden or empty... */
```

Comment:
I think you shouldn't have removed this `if` completely. We still need `repoter-bugzilla` to check that Product is configured properly.